tiny_obj_loader.h

```diff
@@ -654,6 +661,871 @@ bool ParseTextureNameAndOption(std::string *texname, texture_option_t *texopt,
 #include <sstream>
 #include <utility>
 
+#ifdef TINYOBJLOADER_USE_MAPBOX_EARCUT
+
+#include <algorithm>
+#include <array>
+
+/*
+ISC License
+
+Copyright (c) 2015, Mapbox
```

Comment:
Could I ask you to incorporate this new license text into the top-level LICENSE file in this repository (so that there are two license stanzas in that one file)?

I would like to redistribute object code that uses tinyobjloader, so I'll need to incorporate this license text into my redistributed notices.  Doing so is a little bit annoying if the license text is embedded into the C++ header, instead of as a separate file.

Reply:
I see. It'd be better to separate mapbox earcut file into a separate file. Will create PR soon. Also we'll add ISC license to `LICENSE` file.